### ISSUE TITLE
[Flixhq] No stream links are returned for vidcloud and upcloud

### ISSUE THREAD

**Issue by pitsi** [consumet/api.consumet.org#159]
### Describe the bug

The api does not return streams for vidcloud and upcloud. It does return streams for mixcloud though.

### Steps to reproduce

1. Using the search function, find the movie's id or `mediaId` as the api names it. E.g. for deadpool 
```
$ curl -s https://api.consumet.org/movies/flixhq/deadpool | jq .
    {
      "id": "movie/watch-deadpool-19694",
    },
```
2. Using the movie id from above, find the id number for the movie or `episodeId` as the api names it.
```
$ curl -s https://api.consumet.org/movies/flixhq/info?id=movie/watch-deadpool-19694 | jq .
    {
      "id": "19694",
    }
```
3. Using the 2 outputs from above, request the source urls (streams) for vidcloud or upcloud.
```
$ curl -s "https://api.consumet.org/movies/flixhq/watch?episodeId=19694&mediaId=movie/watch-deadpool-19694&server=vidcloud" | jq .

$ curl -s "https://api.consumet.org/movies/flixhq/watch?episodeId=19694&mediaId=movie/watch-deadpool-19694&server=upcloud" | jq .
```

### Expected behavior

The last step should return results similar to this one, which is for mixdrop.
```
$ curl -s "https://api.consumet.org/movies/flixhq/watch?episodeId=19694&mediaId=movie/watch-deadpool-19694&server=mixdrop" | jq .
    {
      "url": "https://s-delivery25.mxdcontent.net/v/0d097bb93c197127f37e24d151e5ff26.mp4?s=2aWSU0kHPEWTnEV_wNqLOQ&e=1665692979&_t=1665677379",
    }
```

### Actual behavior

Instead, a "media not found" message is returned for vidcloud and upcloud.
```
$ curl -s "https://api.consumet.org/movies/flixhq/watch?episodeId=19694&mediaId=movie/watch-deadpool-19694&server=vidcloud" | jq .
{
  "message": "Media Not found."
}

$ curl -s "https://api.consumet.org/movies/flixhq/watch?episodeId=19694&mediaId=movie/watch-deadpool-19694&server=upcloud" | jq .
{
  "message": "Media Not found."
}
```

### Additional context

I don't know when the api broke, but I am sure I used it successfully on a script 2 days ago.
Also, all command outputs above have been trimmed to show only the important part of each one.

**Comment by riimuru** [comment]
@pitsi fixed 👍.

**Comment by pitsi** [comment]
Thank you very very much! :)

**Comment by hackedXD** [comment]
@riimuru This issue seems to have popped up again.

I tried the process done above and it seems to be replicating the same response.

**Comment by pitsi** [comment]
It's almost 7am here and I have just woken up, but I had to check it.

So, on my first try with just curl, the api returned stream links and I was about to say that it was working ok and I would commend later on, with some grep etc to keep the important parts.
Then I tried my script, but it did not work, so I tried curl again. And this time curl returned the "media nor found" message.
I will investigate it more later on, just in case the api has some downtime.

I also noticed that it no longer returns the same urls for vidcloud and upcloud, like it did 2 days ago.
